lib/signs_ui/config/expiration.ex

```diff
@@ -59,6 +59,59 @@ defmodule SignsUi.Config.Expiration do
     |> Enum.into(%{})
   end
 
+  @spec expire_signs_via_alert(list(SignsUi.Config.State.t()), MapSet.t()) ::
```

Comment:
Should this have the same shape as `expire_signs_via_time`?

Reply:
I don't think so. Here's what @losvedir said he wanted in Slack messages to me.
"a pure function called `expire_signs_via_alerts(sign_state, alert_ids)` where the alert_ids are the MapSet above, and the sign_state looks like: `[%SignsUi.Config.Sign{...}, ...]` ...  `expire_signs_via_alerts([signs...], alert_ids)` should return a list of the signs that have an alert_id which is not present in alert_ids. and those signs should be new signs with the same ID, but in auto mode"